libs/ui-components/src/components/Device/EditDeviceWizard/steps/ApplicationTemplates.tsx

```diff
@@ -45,6 +48,7 @@ const ApplicationSection = ({ index, isReadOnly }: { index: number; isReadOnly?:
       setValue(
         {
           specType: AppSpecType.INLINE,
+          appType: (app.appType || ('compose' as unknown as AppForm['appType'])),
```

Comment:
Seems like it would work without the casting
```suggestion
          appType: app.appType,
```